### PR TITLE
Full screen ahead

### PR DESCRIPTION
**Pull request by donokuda**
From https://github.com/desktop/desktop/issues/1809

> Second, I think a notification similar to that of the zoom notification that just landed in #1720 could help out as well.

This PR adds a notification popup whenever a user toggles on full screen mode informing how to get out of full screen mode.

When the user is on macOS, the keys displayed are `⌃⌘F` and on Windows, the keys displayed are `F11`.

#### Couple of gifs demonstrating this feature:

**Windows**
full-screen

**macOS**
output

@niik I'm having an issue trying to get the CSS animation to work when the notification is supposed to go away. Currently it just disappears all at once. I'm open to any suggestions on how to get that working.

### CODE REVIEW

**Comment by niik** [comment]
@donokuda I sorted out the immediate-dissapear thing and made some minor tweaks to the layout, have a look and see what you think. I'm not gonna review this since I've now touched it. I really think we should combine zoom-info and fullscreeninfo into one central component for showing toast notifications but I don't want to hold this up any longer so we can tackle that in a separate PR

**Comment by donokuda** [comment]
Tested on Windows and it looks good to me.

File: app/src/ui/window/full-screen-info.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+interface IFullScreenInfoState {
+  readonly renderInfo: boolean
+  readonly renderTransitionGroup: boolean
+}
+
+const transitionAppearDuration = 100
+const transitionLeaveDuration = 250
+const holdDuration = 3000
+
+export class FullScreenInfo extends React.Component<
```
Comment (joshaber) [inline app/src/ui/window/full-screen-info.tsx]: 📖 🙏

**Comment by donokuda** [comment]
📖 👀

**Comment by joshaber** [comment]
@niik @donokuda Do we consider this as closing #1809?

**Pull request merged**